userservice/src/main/java/com/infodation/userservice/services/UserServiceImpl.java

```diff
@@ -21,6 +29,51 @@ public class UserServiceImpl implements IUserService {
     public UserServiceImpl(UserRepository userRepository) {
         this.userRepository = userRepository;
     }
+    @Async  // Annotation cho bất đồng bộ
+    public CompletableFuture<Void> importUsersFromCsv(MultipartFile file) throws IOException {
```

Comment:
Again, please change the method name to be readable
```suggestion
    public CompletableFuture<Void> importUsersFromCsvAsync(MultipartFile file) throws IOException {
```